tests/installation/module_registration/register_extensions_and_modules.pm

```diff
@@ -11,12 +11,19 @@
 use base 'y2_installbase';
 use strict;
 use warnings;
-use testapi qw(save_screenshot get_required_var);
+use testapi qw(save_screenshot get_required_var get_var);
 
 sub run {
     my @scc_addons = grep($_, split(/,/, get_required_var('SCC_ADDONS')));
     $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
     save_screenshot;
+    $testapi::distri->wait_registration_common_regcode_finished({timeout => 60, interval => 2, message => 'Page to insert module registration code did not appear'});
```

Comment:
you can separate each parameter in a line, `timeout`, `interval`, etc, to make lines a bit more shorter and readable, or at least `message` should be in a different line because it is too long, (just styling, neatpicking :) )